ui-participant/src/studies/enroll/PreEnroll.tsx

```diff
@@ -65,10 +65,37 @@ export default function PreEnrollView({ enrollContext, survey }:
     })
   }
 
+  const fetchProxyQuestion = async (): Promise<string | null> => {
```

Comment:
as mentioned above, this is some clever code that I really hope we don't actually need